optimum/exporters/onnx/model_configs.py

```diff
@@ -142,6 +144,27 @@
     "text2text-generation-with-past",
 ]
 
+COMMON_VLM_TEXT_GENERATION_TASKS = [
+    *COMMON_TEXT_GENERATION_TASKS,
+    "image-text-to-text",
+    "image-text-to-text-with-past",
+]
+
+
+def init_model_configs():
+    """Initialize custom model configs on the task manager."""
+    # Hacky but works, would be cleaner to expose this behavior in the TasksManager
+    TasksManager._CUSTOM_CLASSES[("pt", "gemma3", "image-text-to-text")] = (
+        "transformers",
+        "Gemma3ForConditionalGeneration",
+    )
+    TasksManager._CUSTOM_CLASSES[("pt", "gemma3", "image-text-to-text-with-past")] = (
+        "transformers",
+        "Gemma3ForConditionalGeneration",
+    )
```

Comment:
this is not needed, we can simply add AutoModelForImageTextToText in https://github.com/huggingface/optimum/blob/576942c47c2095820b51f70c2ae888f1ad8e1e9e/optimum/exporters/tasks.py#L166

Reply:
we can keep it this way for now for experimentation but would be better to officially add the task to auto class in optimum at the end